### ISSUE TITLE
Неточные ответы

### ISSUE THREAD

**Issue by 1dylol** [X88Y/mash#8]
Очень многие мне писали, что ответы не точные. Не понимаю, как это может быть, если скрипт это грубо говоря просто дешифратор?

**Comment by X88Y** [comment]
Там где не точные ответы программа предупредит

**Comment by 1dylol** [comment]
> Там где не точные ответы программа предупредит

Нет, ничего не предупреждает. Просто чел кидает джсон, я вставляю, дает ответы, кидаю их ему и он говорит что например 33/100. Причём он не первый, а в скрипте никаких предупреждений нет
